README.md

```diff
@@ -1,273 +1,43 @@
 
-These snapshots are updated on each push to `main`.
+</div>
 
-Weekly snapshots for the Android Studio / IntelliJ plugin [are also available](tree/main/intellij-plugin#weekly-snapshots). 
 
-## Stability of different artifacts
+| ☑️  Apollo Clients User Survey                                                                                                                                                                                                                                                                                                                                                           |
+|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| What do you like best about Apollo Kotlin? What needs to be improved? Please tell us by taking a [one-minute survey](https://docs.google.com/forms/d/e/1FAIpQLSczNDXfJne3ZUOXjk9Ursm9JYvhTh1_nFTDfdq3XBAFWCzplQ/viewform?usp=pp_url&entry.1170701325=Apollo+Kotlin&entry.204965213=Readme). Your responses will help us understand Apollo Kotlin usage and allow us to serve you better. |
 
-Apollo Kotlin is very modular and publishes several artifacts.
+## 🚀 Apollo Kotlin
 
-* Artifacts ending with `-incubating` are not finalized yet and subject to change any time.
-* Other artifacts observe [Semantic Versioning](https://semver.org/).
-    * No breaking change should be introduced in minor or patch releases except for symbols annotated with `@ApolloExperimental` that are subject to change at any time.
-    * Deprecated symbols may be removed in the next major release. We strongly recommend removing deprecated usages before migrating to the next major version.
+Apollo Kotlin is a strongly typed GraphQL client that generates Kotlin models for your GraphQL operations.
 
-## Contributing
+Apollo Kotlin executes operations against a GraphQL server and returns results as operation-specific Kotlin types. This means you don't have to deal with parsing JSON, or passing around `Map`s and making clients cast values to the right type manually. You also don't have to write model types yourself, because these are generated from the GraphQL definitions your app uses.
 
-If you'd like to contribute, please see [Contributing.md](https://github.com/apollographql/apollo-kotlin/blob/main/CONTRIBUTING.md).
+Because generated types are operation-specific, you can only access data that you actually specify as part of an operation. If you don't ask for a particular field in an operation, you can't access the corresponding property on the returned data structure.
 
-## Community integrations
+This library is designed with Android in mind, but you can use it in any Kotlin multiplatform application.
```

Comment:
Nitpick: I wonder if this may be slightly confusing ('can I use it on a SpringBoot backend?'):

```suggestion
This library is designed with Android in mind, but you can use it in any Kotlin application (including multiplatform).
```

Reply:
Yea, good callout. I was trying to avoid the usage of parenthesis, which IMO weakens the message. I'll do that:

```
This library is designed with Android in mind, but you can use it in any Kotlin application, including multiplatform.
```